package.json

```diff
@@ -61,7 +61,8 @@
     "tsd": "^0.28.1",
     "typescript": "^5.0.4",
     "typescript-cached-transpile": "0.0.6",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "v8-profiler-next": "^1.9.0"
```

Comment:
This package was intentionally removed from our package.json here: https://github.com/mongodb/js-bson/commit/409c592524a6eec6a89eb96dee4b51e4f8adf5ae#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L64

It currently fails to install on Node.js 20 (on macos arm at least) so we don't want it to be present by default.

Reply:
Will remove it ! 👍